src/YT/YT_Inline.cpp

```diff
@@ -74,6 +85,14 @@ void YT_Inline()
       NPatchAllLv += NPatchTotal[lv];
 
       GID_LvStart[lv] = ( lv == 0 ) ? 0 : GID_LvStart[lv-1] + NPatchTotal[lv-1];
+
+      // set YT_GID_Offset for searching GID in derived function and particle get attribute function.
+#ifdef LIBYT_USE_PATCH_GROUP
+      if (GID_Offset[lv] % 8 != 0) Aux_Error( ERROR_INFO, "Building search gid YT_GID_Offset in libyt failed !!\n" );
+      YT_GID_Offset[lv] = GID_Offset[lv] / 8;
+#else
+      YT_GID_Offset[lv] = GID_Offset[lv];
+#endif
```

Comment:
We still need that `GID_Offset` at `YT_AddLocalGrid` Line 81 and 120. Though it can somehow be replaced with `YT_GID_Offset`, I think it is more clear to separate them.

Reply:
They are basically the same thing. If you prefer to keep both variables, at least rename one of them to clarify the difference.

Reply:
OK.